addon-scheduler/src/main/java/org/vaadin/stefan/fullcalendar/Resource.java

```diff
@@ -55,11 +54,11 @@ public class Resource {
     private final String color;
 
     /**
-     * The BusinessHours of this resource.
+     * The BusinessHours array of this resource.
      * 
-     * A businessHours declaration that will only apply to this resource.
+     * A businessHours[] declaration that will only apply to this resource.
      */
-    private final BusinessHours businessHours;
+    private final BusinessHours[] businessHoursArray;
     
```

Comment:
Renaming the businessHours to businessHoursArray leads to a breaking change, since the auto generated getter `public BusinessHours getBusinessHours()` will be lost. Please add a custom getter, that restores the old functionality (e.g. if business hours array has at least one item, return the first item, otherwise null).